### DISCUSSION TITLE
Any idea detecting 3D printer failure?

### DISCUSSION THREAD

**Discussion by t0ny-peng** [blakeblackshear/frigate#2173] [Ideas]
Disasters sometimes happen like this. It would be nice to detect that with Frigate and stop the 3D printer immediately. Does anyone have any existing model?

![image](https://user-images.githubusercontent.com/4816327/140014902-8e5c2e60-68ed-4c92-b657-e3da02f6537f.png)

**Comment by patdemko** [comment]
This isn't involving frigate, but is trying to accomplish a similar goal so thought you might be interested.

https://plugins.octoprint.org/plugins/thespaghettidetective/

**Comment by Lukas-Heiligenbrunner** [comment]
One could try to use their docker image with the detection model inside and connect it somehow with frigate:
https://github.com/TheSpaghettiDetective/obico-server/blob/release/ml_api/Dockerfile
